src/ngraph_encapsulate_op.cc

```diff
@@ -582,8 +630,12 @@ class NGraphEncapsulateOp : public OpKernel {
   string m_op_backend_name;
   std::list<std::string> m_lru;
   int NGRAPH_TF_FUNCTION_CACHE_ITEM_DEPTH = 16;
+  static int instance_id_;
+  int my_instance_id_{0};
```

Comment:
Should we rename my_instance_id_ to m_instance_id, keeping with the other member variables?

Same thing for the member variable NGRAPH_TF_FUNCTION_CACHE_ITEM_DEPTH, should it be named to m_cache_depth?